message-queue-lib/src/cmr/message_queue/queue/sqs.clj

```diff
@@ -4,71 +4,72 @@
   comments here. Topics in SNS are (mostly) equivalent to exchanges in RabbitMQ."
   (:require
    [cheshire.core :as json]
-   [clj-http.client :as client]
-   [clj-time.core :as t]
-   [clj-time.format :as f]
    [clojure.core.async :as a]
    [clojure.string :as string]
    [cmr.common.config :as cfg :refer [defconfig]]
    [cmr.common.dev.record-pretty-printer :as record-pretty-printer]
    [cmr.common.lifecycle :as lifecycle]
    [cmr.common.log :as log :refer [debug info warn error]]
-   [cmr.common.mime-types :as mt]
-   [cmr.common.services.errors :as errors]
-   [cmr.common.services.health-helper :as hh]
-   [cmr.common.util :as u]
    [cmr.message-queue.config :as config]
-   [cmr.message-queue.queue.queue-protocol :as queue-protocol]
-   [cmr.message-queue.services.queue :as queue])
+   [cmr.message-queue.queue.queue-protocol :as queue-protocol])
   (:import
    (clojure.lang Keyword Reflector)
    (cmr.message_queue.test ExitException)
-   (com.amazonaws ClientConfiguration)
-   (com.amazonaws.auth BasicAWSCredentials)
-   (com.amazonaws.auth.policy Condition Policy Principal Resource Statement Statement$Effect)
-   (com.amazonaws.auth.policy.actions SQSActions)
-   (com.amazonaws.auth.policy.conditions ConditionFactory)
-   (com.amazonaws.regions InMemoryRegionImpl)
+   ;(com.amazonaws ClientConfiguration)
```

Comment:
if we are not using these imports, why not just delete, instead of commenting them out?

Reply:
yes, that is my plan (which was to subtle with my use of one ; instead of two). I was not sure if this was actually needed being an import and not a require. Tests pass, so I'll remove now.